### ISSUE TITLE
Some questions about reproducing the results in the paper

### ISSUE THREAD

**Issue by qq1332427275** [sunlicai/MAE-DFER#1]
Hello author! ~ I really like your work, fantastic job, I would like to try to perform a reproduction of the results in the paper, such as the dfew dataset, can you explain how to do this, such as how to run it? What parameters should be modified, I see that the input image size in the code is 160, but the dfew dataset Clip is 224*224, what kind of preprocessing is done to the data? Thank you very much for reading ~ looking forward to your reply!

**Comment by sunlicai** [comment]
Hi, thanks for your interest in our work! 

For fine-tuning on DFEW, you can check the running script [here](https://github.com/sunlicai/MAE-DFER/blob/master/scripts/dfew/finetune_local_global_attn_depth16_region_size2510_with_diff_target_164.sh). The hyperparameters in the script shall be the same as that given in the paper. You can also modify them (e.g., batch size and learning rate) as you want and you might achieve better results as we do not heavily tune them. 

For facial image size, since the original size of cropped face provided by DFEW is 256\*256 (not 224\*224), we [resize it to 160*160](https://github.com/sunlicai/MAE-DFER/blob/f9ee9cb8f425f24540dae1d66d32ed89212fa31f/kinetics.py#L336C40-L336C40
) before patch embedding. More augmentations can be found in the dataset code.

**Comment by qq1332427275** [comment]
Okay! I understand! Thank you very much~ :D

**Comment by qq1332427275** [comment]
@sunlicai 
I have some small problems with the model, can you please answer them, it won't take much of your time.
1. my dataset is not balanced and I want to use a different loss function (e.g. focal loss) do I just need to change criterion to the new loss function and turn off mixup_fn (if I don't turn it off it will give me an error)
2. does this code support regression task, my dataset needs to perform (0,1) regression task, if so how to modify it?
Looking forward to your reply!

Translated with www.DeepL.com/Translator (free version)

**Comment by qq1332427275** [comment]
@sunlicai 
I have some small problems with the model, can you please answer them, it won't take much of your time.
1. my dataset is not balanced and I want to use a different loss function (e.g. focal loss) do I just need to change criterion to the new loss function and turn off mixup_fn (if I don't turn it off it will give me an error)
2. does this code support regression task, my dataset needs to perform (0,1) regression task, if so how to modify it?
Looking forward to your reply!

**Comment by youcaiSUN** [comment]
Hi, my answer is as follows:

1. Yes!
2. Currently, the code does not support regression task. However, I believe that the modification is simple, i.e., you just use your own regression loss (e.g., mse or l1) as in question (1).

Best,
Licai
